### DISCUSSION TITLE
Github education

### DISCUSSION THREAD

**Discussion by nguyenhieu2k3** [community/community#146886] [GitHub Education]
### Hi everyone,

I registered for github education on December 6 and was notified that I was approved, but now it's Friday and I still haven't received an email.

**Comment by himusharier** [comment]
Hi there, congratulations on getting approved by github education.
Generally you should get one email as confirmation. In my case, I found my confirmation email in spam (don't know why!).
You should check your spam mailbox again. 
Otherwise, you can check your status at: https://education.github.com/discount_requests/application?type=student
(check if you are approved or not)
![image](https://github.com/user-attachments/assets/fb4d28c1-de1f-496d-baf4-21f63a90effe)

Then, checkout this link (https://education.github.com/pack), scroll down a bit and check if you can avail their offers.

**Comment by davevad93** [comment]
Hi @nguyenhieu2k3 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @nguyenhieu2k3 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @nguyenhieu2k3 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #144741, #145995
